elections/README.md

```diff
@@ -0,0 +1,80 @@
+* [Kata Containers Elections](#kata-containers-elections)
+  * [Architecture Committee Elections](#architecture-committee-elections)
+      * [Election Dates](#election-dates)
+      * [Electorate](#electorate)
+      * [Candidacy](#candidacy)
+      * [Architecture Committee Responsibilities](#architecture-committee-responsibilities)
+      * [System](#system)
+
+
+
+# Kata Containers Elections
+
+This repository contains Kata Containers Elections reference documents
+and tooling to run elections.
+
+
+# Architecture Committee Elections
+
+
+## Election Dates
+
+The Architecture Committee elections take place every 6 months, with 3 seats
+up for election in September and 2 seats up for election in February.
+
+The election process begins 3 weeks prior to the election.
+- One week for candidates to submit their candidacy
+- One week "debate period" for electorate to pose questions on the community
+mailing lists to learn the candidates' platforms
+- One week for voting
+- Results announced
+
+## Electorate
+
+Contributors (defined as anyone who has had code merged in the Kata Containers
+project in the last 12 months) are eligible to vote in Architecture
+Committee elections.
+
+## Candidacy
+
+Anyone who has made contributions to the Kata Containers project--code or
+otherwise--is eligible to run. Candidates submit their candidacy by adding
+a textfile with their qualifications, interest, and platform to the
+appropriate election folder under the community repo, and announcing their
+candidacy on the kata-dev mailing list.
+
+There are no term limits for Architecture Committee seats, but in order to
+encourage diversity, no more than 2 of the 5 seats may be filled by members
+with the same company affiliation. If a third member would be in violation
+of the diverse affiliation rule, the seat is awarded to the next highest
+scoring candidate who would not violate the rule. Members who change
+affiliation between elections in a way that violates the diverse affiliation
+rule are not required to relinquish their seat, however, the rule will
+be enforced and applied in the following election.
+
+## Architecture Committee Responsibilities
+
+Architecture Committee members guide the technical direction of the Kata
+Containers project. This includes, but is not limited to:
+- Serving the Kata technical community
+    - Driving technical direction
+    - Handling interactions with other governance bodies
+    - Acting as the final call in technical disputes
+    - Ensuring inclusive, open collaboration and development
+    - Identifying gaps in technical direction or community health
+- Attending Architecture Committee meetings
+    - Regular participation (and advanced notice of absences) in online
+    Arch Committee meetings and in-person meetings is expected
+- Actively participates in Kata Containers design and development
+    - Not restricted to code contributions, but active input and guidance
+    is expected
```

Comment:
Since the Architecture Committee has significant responsibility, should we also document the process to follow if a member is unable to fulfill these responsibilities or is deemed to not be fulfilling them? Again, that might be best in the community README?

Reply:
Since terms are only a year, is it sort of a self-cleaning process in which members that aren't fulfilling their duties are likely not going to be voted-in again by the community?

Reply:
Sure. But a year is a long time on this project ... ;)